core/block_validator.go

```diff
@@ -93,6 +98,12 @@ func (v *BlockValidator) ValidateBody(block *types.Block) error {
 			return r
 		}
 	}
+	// for fast node which verify trie from remote verify peers, a block's H-11 ancestor should have been verify.
+	if v.remoteValidator != nil {
+		if !v.remoteValidator.AncestorVerified(v.bc.GetHeaderByNumber(header.Number.Uint64())) {
+			return fmt.Errorf("block's ancessor %x has not been verified", block.Hash())
+		}
+	}
```

Comment:
Wrap these code as a new validateFunc is better?

Reply:
Please run concurrently like another two validateFuncs.